src/components/playVideo/PlayVideo.jsx

```
@@ -1075,27 +1135,31 @@ const PlayVideo = ({ videoDetails, author, permlink, playlistData, onClosePlayli
-                >
-                  <Repeat2 size={20} />
-                </button>
-              </div>
+              {authenticated && isLoggedIn() && (
+                <div className="fab-action">
+                  <span className="fab-action-label">Reshare{reshareCount > 0 ? ` (${reshareCount})` : ''}</span>
+                  <button
+                    className={`fab-action-btn${hasReshared ? ' fab-action-btn--voted' : ''}`}
+                    onClick={() => { handleReshare(); setFabOpen(false); }}
+                    aria-label="Reshare"
+                  >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)